addon/mixins/legend.js

```diff
@@ -49,12 +49,15 @@ export default Ember.Mixin.create({
     }
   }),
 
-  // Dynamically calculate the number of legend items in each row
+  // Dynamically calculate the number of legend items in each row.
+  // This is only an approximate value to estimate the maximum required space for legends
   numLegendItemsPerRow: Ember.computed('legendWidth', 'legendItemWidth', function() {
-    return Math.floor(this.get('legendWidth') / this.get('legendItemWidth'));
+    // Since we are using Math.floor, make sure we always return 1 as the least value.
```

Comment:
Assume that the people reading your comments are literate in JavaScript, don't just describe what code is doing, but why it's doing it. Why do we care that this item is at least 1? Also, this should probably be in the documentation comment above.